README.md

```diff
@@ -14,74 +17,78 @@
+### 🌿 Backend Code
 
-### 🌿 An OpenAPI spec
+Define your API, and Fern will generate models, endpoints and boilerplate application code. All the generated code is validated at build time, ensuring your all your endpoints and their data are being served correctly.
 
-You can feed the generated OpenAPI into the endless list of tools that support OpenAPI.
+We currently supported:
 
-# Comparison with OpenAPI
+- Express
+- Spring Boot
+- FastAPI
 
-OpenAPI is a great tool for documenting APIs, but falls short for code generation use cases.
+### 🌿 Documentation
 
-Read [Fern vs. OpenAPI](https://buildwithfern.com/docs/comparison-with-openapi)
-for an in-depth comparison. **TL;DR: we differ from OpenAPI in these areas:**
+Fern provides first class support for generating documentation, ranging from [fully features documentation websites](https://docs.vellum.ai/api-reference/generate) to Postman Collections. This allows you to always keep your documentation, API and Postman collections in sync with ease.
+
+## Fern and OpenAPI
+
+We have designed Fern to complement existing OpenAPI toolchains and workflows, rather than replace them. We believe OpenAPI does a good job at as a declarative standard for defining APIs, but at times it is too verbose and complex, which harms the quality of automatically generated code.
+
+The goal of Fern is to be **complimentary to OpenAPI**, not to replace it. As such, you are never locked in to Fern, its specifications, or its tools.
+
+If you want to read more about where Fern and OpenAPI differ, read [﻿Fern vs. OpenAPI](https://buildwithfern.com/docs/comparison-with-openapi) for an in-depth comparison.
+
+**TL;DR: we differ from OpenAPI in these areas:**
 
-- [Simplicity](https://www.buildwithfern.com/docs/comparison-with-openapi#simplicity)
 - [New features in specification](https://www.buildwithfern.com/docs/comparison-with-openapi#new-features-in-specification)
 - [Quality of code generation](https://www.buildwithfern.com/docs/comparison-with-openapi#quality-of-code-generation)
 - [Focus on server-side API development](https://www.buildwithfern.com/docs/comparison-with-openapi#focus-on-server-side-api-development)
 - [Change management](https://www.buildwithfern.com/docs/comparison-with-openapi#change-management)
 - [Cloud-based code generation and publishing](https://www.buildwithfern.com/docs/comparison-with-openapi#cloud-based-code-generation-and-publishing)
 
-# Plant Store example
-
-Plant Store is full example of an API defined in Fern.
-
-- [API Definition](https://github.com/fern-api/plantstore-api)
-- [Generated TypeScript SDK](https://github.com/fern-api/plantstore-node)
-- [Generated Java SDK](https://github.com/fern-api/plantstore-java)
-- [Generated Postman Collection](https://github.com/fern-api/plantstore-postman)
-- [Generated OpenAPI spec](https://github.com/fern-api/plantstore-openapi)
+# Getting Started
 
-# Get started
+The Fern tools are available as a Node package. To install it, simply run:
```

Comment:
if we are going to have a CLI Reference at the bottom of the readme then i think we should be consistent with terminology -- so let's use CLI instead of `Node package`

Reply:
Agreed.